src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
I've been told that Firefox should already have flags to check for the existence of rotation/position via a flag on the Gamepad handler: `gamepad.hasOrientation / gamepad.hasPosition`